### PR TITLE
[vnc] Support for multiple users. Fixes JB#47602

### PR DESCRIPTION
**Pull request by mkenttala**

### CODE REVIEW

**Comment by mkenttala** [comment]
That currentuser link creation should probably be moved to some more generic packages as it could be useful elsewhere too.

File: data/make_user_link.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+USER_UID=$(loginctl list-sessions | grep seat0 | tr -s " " | cut -d " " -f 3)
+ln -fs /run/user/$USER_UID /run/currentuser
```
Comment (rainemak) [inline data/make_user_link.sh]: Should this be part of sessions package?

Comment (rainemak) [reply data/make_user_link.sh]: Current or active ?

Systemd wise it should be matched to the active user.

Comment (mkenttala) [reply data/make_user_link.sh]: Link creation moved jolla-sessions-qt5 and renamed to activeuser

Comment (rainemak) [reply data/make_user_link.sh]: Ok

**Comment by mkenttala** [comment]
No write access so cannot merge

**Pull request merged**